bindings/wasm/bindings.js

```diff
@@ -142,6 +143,7 @@ Module.setup = function () {
       this.originalID = originalID;
       this.faceID = faceID;
       this.halfedgeTangent = halfedgeTangent;
+      this.transform = transform;
```

Comment:
It occurs to me I haven't really followed my own naming convention here - I should switch `originalID` to `runOriginalID` and `transform` to `runTransform`, since I generally name with what I'm indexing from and then to.